test/integration/orphan_finder_test.go

```diff
@@ -0,0 +1,116 @@
+// +build integration
+
+package integration
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"math/big"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+)
+
+var template = `[AUDIT] Failed RPC to store at SA, orphaning precertificate: serial=[%x] cert=[%x] err=[sa.StorageAuthority.AddCertificate timed out after 5000 ms], regID=[1], orderID=[1]
+[AUDIT] Failed RPC to store at SA, orphaning certificate: serial=[%x] cert=[%x] err=[sa.StorageAuthority.AddCertificate timed out after 5000 ms], regID=[1], orderID=[1]`
+
+// TestOrphanFinder runs the orphan-finder with an example input file. This must
+// be run after other tests so the account ID 1 exists (since the inserted
+// certificates will be associated with that account).
+func TestOrphanFinder(t *testing.T) {
+	precert, err := makeFakeCert(true)
+	if err != nil {
+		log.Fatal(err)
+	}
+	cert, err := makeFakeCert(false)
+	if err != nil {
+		log.Fatal(err)
+	}
+	f, _ := ioutil.TempFile("", "orphaned.log")
+	io.WriteString(f, fmt.Sprintf(template, precert.SerialNumber.Bytes(),
+		precert.Raw, cert.SerialNumber.Bytes(), cert.Raw))
+	f.Close()
+	out, err := exec.Command("../../bin/orphan-finder", "parse-ca-log",
+		"--config", "../../"+os.Getenv("BOULDER_CONFIG_DIR")+"/orphan-finder.json",
+		"--log-file", f.Name()).Output()
```

Comment:
I think you have some path issues here that are causing CI to fail:
```
--- FAIL: TestOrphanFinder (0.57s)
    orphan_finder_test.go:46: orphan finder failed ("exit status 1"). Output was: E000809 orphan-finder lrGdwwk [AUDIT] Failed to read config file: open ../..//orphan-finder.json: no such file or directory
FAIL
FAIL	github.com/letsencrypt/boulder/test/integration	0.575s
FAIL
```

(Cosmetically there's also a duplicated `//` in the produced path: `../..//orphan-finder.json`)

Reply:
Weird. I'll look into that. I think what's actually going on is that `BOULDER_CONFIG_DIR` is not being passed through or is somehow evaluating to the empty string.